src/clean-up.ts

```diff
@@ -0,0 +1,22 @@
+import { PlainClientAPI } from 'contentful-management';
+import { initClient } from './client/init-client';
+import { cleanUpTestEnvironments } from './environment/clean-up-test-environments';
+
+type CleanUpSpacesOptions = {
+  client?: PlainClientAPI;
+  spaces: string[];
+  regex?: RegExp;
+  threshold?: number;
+  dryRun?: boolean;
+};
+
+export async function cleanUp(options: CleanUpSpacesOptions) {
+  const client = options.client ?? initClient();
+  for (const space of options.spaces) {
+    await cleanUpTestEnvironments({
```

Comment:
Should this maybe first cleanup all test spaces, and then all test environments on the spaces that are still around? 🤔